test/jest/integration-capture.spec.ts

```diff
@@ -0,0 +1,132 @@
+import IntegrationCapture from "../../src/integrationCapture";
+
+function deleteAllCookies() {
+    document.cookie.split(';').forEach(cookie => {
+        const eqPos = cookie.indexOf('=');
+        const name = eqPos > -1 ? cookie.substring(0, eqPos) : cookie;
+        document.cookie = name + '=;expires=Thu, 01 Jan 1970 00:00:00 GMT';
```

Comment:
is this best practice? not sure, seems funky

Reply:
is setting the expires time to the past a shortcut for deleting a cookie?

Reply:
Yeah, you'd think Jest would have a better way of mocking or clearing cookies. 😦